proto/message_contents/invitation.proto

```diff
@@ -19,13 +19,19 @@ message InvitationV1 {
         bytes key_material = 1;  // randomly generated key material (32 bytes)
     }
 
+    message GroupContext {
```

Comment:
tl/dr: I would avoid the word "Group" here entirely.
The "group" metaphor (to me at least) implies a set of "members" who can be listed/added/removed etc. But if I understand correctly, this won't support listing or removing members. Having a field called "GroupContext" seems like a footgun ("oh cool, XMTP supports group messaging, how do I build the group list? ah! `GroupContext` I'll use that to show the member list").  "Conversation" already implies (to me at least) the possibility of multiple participants.